lib/turkish_cities/distance.rb

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+
+require 'i18n'
+require 'yaml'
+
+require_relative '../turkish_cities/helpers/decomposer_helper'
+
+class Distance
+  include DecomposerHelper
+
+  file_path = File.join(File.dirname(__FILE__), 'data/cities.yaml')
+  CITY_LIST = YAML.load_file(file_path)
+
+  def distance_between(from, to, travel_method)
```

Comment:
bu `from` ve `to`'yu her method'da vermek yerine class'i initlesek ve orada attribute olarak olsa mesela.

```ruby
distance = Distance.new('Istanbul', 'Ordu')
distance.distance_between_air
```

gibi olsa mi?